lib/doc/worksheet.js

```diff
@@ -782,8 +783,32 @@ class Worksheet {
     this._media = value.media.map(medium => new Image(this, medium));
     this.sheetProtection = value.sheetProtection;
     this.tables = value.tables.reduce((tables, table) => {
-      const t = new Table();
-      t.model = table;
+      /**
```

Comment:
Should all of this logic move into Table? it isn;t Worksheet logic.

Reply:
Do we have access to all the cells from a Table? If so I'm fine with that, just let me know where do you want it in there.